src/test/resources/application-tests.properties

```diff
@@ -0,0 +1,8 @@
+spring.h2.console.enabled=true
+spring.datasource.url=jdbc:h2:~/myDB
+spring.h2.console.path=/h2-console
+spring.datasource.username=sa
```

Comment:
`sa`? Why such username?

Reply:
Because it is default value and I didn't change it yet :)